precompile/allowlist/config_test.go

```diff
@@ -0,0 +1,97 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package allowlist
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVerifyAllowlistConfig(t *testing.T) {
+	admins := []common.Address{{1}}
+	enableds := []common.Address{{2}}
+	tests := []struct {
+		name          string
+		config        AllowListConfig
+		expectedError string
+	}{
+		{
+			name:          "invalid allow list config in tx allowlist",
```

Comment:
Can we remove tx allowlist from the names of these tests?

Reply:
removed